src/popup/pages/Dashboard.vue

```diff
@@ -13,13 +13,14 @@
       <LatestTransactionsCard />
 
       <DashboardCard
-        v-if="IS_CORDOVA"
+        v-if="IS_CORDOVA || UNFINISHED_FEATURES"
         :title="$t('dashboard.daeppBrowserCard.title')"
         :description="$t('dashboard.daeppBrowserCard.description')"
         :btn-text="$t('dashboard.daeppBrowserCard.button')"
         :background="daeppBrowserBackground"
         :icon="GlobeIcon"
         :to="{ name: ROUTE_APPS_BROWSER }"
+        :card-id="DASHBOARD_CARD_ID.daeppBrowser"
```

Comment:
I believe this was intentionally left out, because we don't want this card to be dismissable. @ZGiovas?

Reply:
yes @martinkaintas is right

Reply:
OK, I consulted this also with Paolo. I suggest to leave this line but to pass `null` from the constant. Here's the change with comment: https://github.com/aeternity/superhero-wallet/pull/2305/files#diff-9b9a49cd15349e68109f9c49f06bbda53dc9f38b4885771124ee55a2cf0dbf27R367
What do you think?

Reply:
Looks good to me